deployment/deploy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+UPSTREAM_URL="https://github.com/google/osv.dev"
+UPSTREAM_REMOTE_NAME="upstream-release"
+
+# Return if the deployment time is suboptimal
+oss_fuzz_time() {
+  # http://go/osv-deploy says not to deploy between 12am and 3am
+  current_hour=$(TZ=Australia/Sydney date +%-H) # '-' to remove leading 0 which will be parsed as hex
+  if [[ $current_hour -ge 0 && $current_hour -lt 3 ]]; then
+    return 1
+  fi
+  return 0
+}
+
+if [ $# -lt 1 ]; then
```

Comment:
Let's make this file consistently follow https://google.github.io/styleguide/shellguide.html#test----and---

```suggestion
if [[ $# -lt 1 ]]; then
```